docs/websites/erp/install-ofbiz-with-nginx-postgresql/index.md

```diff
@@ -0,0 +1,515 @@
+
+        $ ./gradlew cleanAll loadAll
+
+5.  Start OFBiz. This will print a lot of informations to your console, so suppress them and run on the background:
+
+        $ ./gradlew ofbiz &> /dev/null & disown
+
+    Alternatively, start OFBiz on the background:
+
+        $ ./gradlew ofbizBackground
+
+6.  Navigate to your OFBiz administration interface on `https://your_linode_ip:8443/webtools`. Since OFBiz use self-signed SSL certificate, certificate warning will be thrown by your browser. Bypass the warning and proceed anyway.
+
+    You should now see the page which invites you to log in:
+
+    [![Web Tools interface, not logged in](ofbiz-webtools-main-page-login_small.png "Web Tools interface, not logged in")](ofbiz-webtools-main-page-login.png)
+
+    Click login link and you will see log in page:
+
+    ![Log in page](ofbiz-login.png)
+
+    Log in to OFBiz with `admin` username and `ofbiz` password. You should be landed on main page of *Web Tools* administration interface, which look like:
+
+    [![Web Tools interface, logged in](ofbiz-webtools-main-page-interface_small.png "Web Tools interface, logged in")](ofbiz-webtools-main-page-interface.png)
+
+    Now you can explore the rests of OFBiz interface to get acquainted.
+
+7.  When you have done using OFBiz, stop the instance:
+
+        $ ./gradlew "ofbiz --shutdown"
+
+## Plugins
+
+With Plugins, one can extend OFBiz using pre-built solutions, without having to reinvent them. OFBiz developers maintain official plugins which [the full list can be checked here](https://cwiki.apache.org/confluence/display/OFBIZ/Plugins+stack).
+
+Below `scrum` plugin will be installed. This plugin allow enterprises to manage agile development activities. You can substitute `scrum` for other plugins that you interested. The `pullPluginSource` task will download, build, and install the plugin from source.
+
+To install the plugin:
+
+    $ ./gradlew pullPluginSource -PpluginID=scrum
+
+{{< note >}}
+You need to have recent version of Subversion installed on your system before installing any plugins using `gradlew` script.
+{{< /note >}}
+
+Alternatively, to install all plugins at once:
+
+    $ ./gradlew pullAllPluginsSource
+
+You can also download all plugins using Git:
+
+    $ git clone --depth 1 https://github.com/apache/ofbiz-plugins.git plugins
+
+For the plugins to be effective, rebuild OFBiz:
+
+    $ ./gradlew cleanAll loadAll
+
+Next time when OFBiz is started (`./gradlew ofbiz`), interface for each plugins's component (`scrum` for example) can be accessed via web interface, like:
+
+[![Scrum plugin interface](ofbiz-scrum_small.png "Scrum plugin interface")](ofbiz-scrum.png)
+
+## Serve OFBiz with NGINX
+
+OFBiz instances can be served by using reverse proxy. Here we will use NGINX, which is suitable for this task (fast, lightweight, etc.). For more details of NGINX configurations refer to [Getting Started with NGINX guide series](docs/web-servers/nginx/nginx-installation-and-basic-setup/).
```

Comment:
Hi @bagasme -

I think I found the problem on this line. The relevant docs404 error log lines are here:
https://travis-ci.org/linode/docs/jobs/609578691#L327-L330

The URL that it can't find is:
http://localhost:1313/docs/websites/erp/install-ofbiz-with-nginx-postgresql/docs/web-servers/nginx/nginx-installation-and-basic-setup/

This line has a relative link: `docs/web-servers/nginx/nginx-installation-and-basic-setup/`. Hugo interprets that as linking to a file inside the current guide's directory, so it compiles the link as `docs/websites/erp/install-ofbiz-with-nginx-postgresql/docs/web-servers/nginx/nginx-installation-and-basic-setup/`.

To fix this, insert a `/` at the front of the link:

`[Getting Started with NGINX guide series](/docs/web-servers/nginx/nginx-installation-and-basic-setup/)`